plugins/win-capture/data/locale/en-US.ini

```diff
@@ -48,6 +48,11 @@ Compatibility.WindowCapture.BitBlt="%name% may not be capturable using the selec
 
 # Specific compatibility messages
 Compatibility.Application.CSGO="CS:GO may require the <code>-allow_third_party_software</code> launch option to use Game Capture."
-Compatibility.Application.ElectronGameCapture="Games built on Electron cannot be captured using Game Capture. Use Window Capture or Display Capture instead."
-Compatibility.Application.ElectronBitBlt="Electron-based applications may not be capturable using the selected Capture Method (BitBlt)."
+Compatibility.Application.ChromiumGameCapture="Games built on Chromium cannot be captured using Game Capture. Use Window Capture or Display Capture instead."
+Compatibility.Application.ChromiumBitBlt="Chromium-based applications may not be capturable using the selected Capture Method (BitBlt)."
+Compatibility.Application.UWPBitBlt="UWP applications may not be capturable using the selected Capture Method (BitBlt)."
+Compatibility.Application.UWPGameCapture="UWP applications cannot be captured using Game Capture. Use Window Capture or Display Capture instead."
```

Comment:
This may be misleading, UWP games can be game captured, but I'm not sure if they have a different window class.